xdsl/dialects/llvm.py

```diff
@@ -755,6 +792,127 @@ def get(
         )
 
 
+LLVM_CALLING_CONVS: tuple[str, ...] = (
+    "ccc",
+    "fastcc",
+    "coldcc",
+    "cc 10",
+    "cc 11",
+    "webkit_jscc",
+    "anyregcc",
+    "preserve_mostcc",
+    "preserve_allcc",
+    "cxx_fast_tlscc",
+    "tailcc",
+    "swiftcc",
+    "swifttailcc",
+    "cfguard_checkcc",
+)
+"""
+A list of all valid calling conventions understood by LLVM, see
+https://llvm.org/docs/LangRef.html#calling-conventions
+for more info.
+"""
+
+
+@irdl_attr_definition
+class CallingConventionAttr(ParametrizedAttribute):
+    """
+    LLVM Calling convention, default is ccc.
```

Comment:
Where is the default used?

Reply:
If you have a `func.func` in MLIR (in the default syntax), it will have `CConv` of `ccc`.